### ISSUE TITLE
Putting animate/plot as arguments rather than separate functions to be manually run?

### ISSUE THREAD

**Issue by AlexBradley0** [dascroft/Solving-TDSE-for-an-Arbitrary-V-x-#32]
Would it be better to put the decision of plot vs animate as an argument so that the user only has to run a single line?
Rather than how it currently is where one as to use:
tdse=TDSE(**kwargs)
tdse.plot()

**Comment by AlexVernis** [comment]
If we put it somewhere as an if statement (presumably a new kwarg within the TDSE class, then an if statement in the running box checking what they input for that, and if it's not one of the options print an error message), that could work. But in turn that might make it a required class, it depends.

**Comment by AlexBradley0** [comment]
Pretty much what i was thinking, it wouldnt be required if we set a default of one (probably animate). I'd like it do be able to have the ability to do both at once which might be more complicated but probably not too bad.

Does someone want to go ahead and add that then?  (I'll happily do it)

**Comment by AlexVernis** [comment]
I'm in a lecture right now (and have another right after), I'll leave it to you if you want to get it done right away, otherwise i can do it in like an hour and a half.

**Comment by AlexBradley0** [comment]
I have the same lectures, so either way it'll have to wait

**Comment by AlexBradley0** [comment]
I've done this and it can be merged with main @dascroft   (assuming we're all in agreement with this change?)

For this the kwarg is 'Output', and can currently accept either 'gif' or 'plot'

**Comment by dascroft** [comment]
Opening Pull request

**Comment by dascroft** [comment]
Ah so your removing the argue function? Can the user still specify parameters?

**Comment by AlexVernis** [comment]
without the argue block it won't run with commandline user inputs, but the commandline section is also currently causing issues. (i'm working on it)

**Comment by AlexBradley0** [comment]
Ah, apologies. I only removed that because it was causing crashes so made testing difficult

**Comment by dascroft** [comment]
I see, so the commandline portion will be handled differently to the current setup? Just want to be sure before i commit to main.

**Comment by AlexVernis** [comment]
it will still be about the same i think, it's just kinda buggy right now. it's not playing well with the kwargs.
And presumably the others are still working on making a more intuitive version for user input, commandline just has some issues that need ironing out.

**Comment by AlexVernis** [comment]
tl;dr the handling of it will be about the same, but removing it for now will save headaches.

**Comment by dascroft** [comment]
I see, alright ill merge with main, as i said mentioned on a previous issue, if you need to reference the old code, you can look at the other branches for it or see the commit history, incase it is needed for re-writing the comline interface. With this pull request, this issue is now closed
